app/views/sessions/failure_to_proof.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= render 'shared/alert_error', alert_heading: t('alerts.failure_to_proof.heading'), alert_text: t('alerts.failure_to_proof.text') %>
+
+<div>
+  <h1>What you can do:</h1>
+  <p>
+    For problems with your Login.gov account, please review <a href="https://login.gov/help" target="_blank">Login.gov's Help Section</a>
+    or contact them at <a href="https://login.gov/contact" target="_blank">login.gov/contact</a>.
+  </p>
+  <p>
+    If you've taken steps above and still can't verify your identity, please contact <a href="mailto:team@challenge.gov" target="_blank">team@challenge.gov</a>.
+  </p>
+  <p>
+    You still may have access to the Challenge.gov platform without verifying your identity, but your access will be limited to non-sensitive
+    information. Please proceed to login at the top of the page.
```

Comment:
I think the message is a little misleading because they have already logged in here. Let's remove this last sentence.
```suggestion
    information.
```